apps/shell/public/javascripts/ood_shell.1.js

```diff
@@ -74,7 +72,7 @@ OodShell.prototype.closeTerminal = function (ev) {
     errorDiv.innerHTML = 'Failed to establish a websocket connection. Be sure you are using a browser that supports websocket connections.';
     this.element.appendChild(errorDiv);
   } else {
-    this.term.io.print('\r\nYour connection to the remote server has been terminated.');
+    this.term.io.print('\r\nYour connection to the remote server has been interrupted. Attempting to reconnect.');
```

Comment:
we should probably toggle the visibility of a warning message that is fixed above the terminal, instead of writing to the terminal

Reply:
Mabye onopen adds a class to `<body>` named "connected" and onclose removes this class? Then you could add a message to the dom with css to fix it to the top right or top left that says "Attempting to connect to server..." that `display: none` if "connected" is set on the body.

That way we avoid printing to the term.io.